src/encoded/static/scss/encoded/modules/_key-value-display.scss

```diff
@@ -23,6 +23,9 @@ dl.key-value, dl.key-value-doc {
         width: auto;
         word-wrap: break-word;
         word-break: break-word;
+        -webkit-hyphens: auto;
+        -ms-hyphens: auto;
+        hyphens: auto;
```

Comment:
Do we need the browser prefixes? Seems like all relevant browsers support the `hyphens` property. https://caniuse.com/#search=hyphen

Reply:
Not really requesting changes per se; just _if_ the browser prefixes aren’t needed, seems like you could delete them.

Reply:
Good point. I used caniuse when working on this problem. In order to support Safari, "-webkit-hypens" is need. IE 11 supports hypens only with "-ms-hypens".  Mozilla, Edge and Chrome edge versions work with the standard "hypens". I got this information on caniuse by hovering over each box with a gray dash with a yellow background (far right on certain boxes)  reading what non-standard means is needed to enable support hypens.